app/views/content_items/_print_meta_data.html.erb

```diff
@@ -0,0 +1,33 @@
+<br/>
```

Comment:
This print metadata is specific to HTML publications, can the partial name be clearer?